colorutils.cpp

```diff
@@ -0,0 +1,32 @@
+#include "colorutils.h"
+#include <sstream>
+
+namespace TelCoColorCoder {
+	constexpr int numberOfMajorColors = 5;
+	constexpr int numberOfMinorColors = 5;
+
+	ColorPair GetColorFromPairNumber(int pairNumber)
+	{
+		int zeroBasedPairNumber = pairNumber - 1;
+		MajorColor majorColor =
+			(MajorColor)(zeroBasedPairNumber / numberOfMinorColors);
+		MinorColor minorColor =
+			(MinorColor)(zeroBasedPairNumber % numberOfMinorColors);
+		return ColorPair(majorColor, minorColor);
+	}
+
+	int GetPairNumberFromColor(MajorColor major, MinorColor minor)
+	{
+		return major * numberOfMinorColors + minor + 1;
+	}
+
+	std::string GetFormattedColorCoding() {
```

Comment:
Good approach to make a void function testable - separating the processing/formatting from the I/O 👍 
Now this function can be re-used to output to html too!